pyxrf/model/load_data_from_db.py

```diff
@@ -43,6 +43,46 @@
     pass
 
 
+def flip_data(input_data, subscan_dims=None):
+    """
+    Flip 2D or 3D array. The flip happens on the second index of shape.
+    .. warning :: This function mutates the input values.
+
+    Parameters
+    ----------
+    input_data : 2D or 3D array.
+
+    Returns
+    -------
+    flipped data
+    """
+    new_data = np.asarray(input_data)
+    data_shape = input_data.shape
+    if len(data_shape) == 2:
+        if subscan_dims is None:
+            new_data[1::2, :] = new_data[1::2, ::-1]
+        else:
+            i = 0
+            for nx, ny in subscan_dims:
+                start = i + 1
+                end = i + ny
+                new_data[start:end:2, :] = new_data[start:end:2, ::-1]
+                i += ny
+
+    if len(data_shape) == 3:
+        if subscan_dims is None:
+            new_data[1::2, :, :] = new_data[1::2, ::-1, :]
+        else:
+            i = 0
+            for nx, ny in subscan_dims:
+                start = i + 1
+                end = i + ny
+                new_data[start:end:2, :, :] = new_data[start:end:2, ::-1, :]
+                i += ny
+    return new_data
+
```

Comment:
```suggestion
```

Reply:
I think the permanent fix to formatting issues in PyXRF is to include flake8 check in travis built, but it requires reformatting of all PyXRF code.

Reply:
Agree. It's worth to do it via a separate PR as it's a major update.